Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```
@@ -222,6 +234,11 @@
         prob: 0.3
 #LPP
       - id: LPPClothingOuterDetectiveHardsuit
+      # -< DarkSide
+      - id: LPPBoxFolderSecDoc
+        amount: 2
+        prob: 0.5
+      # - >
```

yamllint

[warning] comment not indented like content

(comments-indentation)